spec/formatting.md

```diff
@@ -445,10 +445,10 @@ and ends with U+007D RIGHT CURLY BRACKET `}`.
 Between the brackets, the following contents are used:
 
 - Expression with Literal Operand: U+007C VERTICAL LINE `|`
-  followed by the value of the Literal,
+  followed by the Quoted or Unquoted value,
```

Comment:
This change is wrong, since you're defining a quoted literal here. I would suggest de-naming Literal here:

```suggestion
  followed by the value of the literal,
```